azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/model/Runtime.java

```diff
@@ -34,6 +37,91 @@
     public static final Runtime LINUX_JAVA11_TOMCAT85 = new Runtime(OperatingSystem.LINUX, WebContainer.TOMCAT_85, JavaVersion.JAVA_11);
     public static final Runtime DOCKER = new Runtime(OperatingSystem.DOCKER, null, null);
 
+    public static Runtime getRuntime(OperatingSystem operatingSystem, WebContainer webContainer, JavaVersion javaVersion) {
+        return values().stream()
+                .filter(runtime -> runtime.operatingSystem == operatingSystem)
+                .filter(runtime -> runtime.webContainer == webContainer)
+                .filter(runtime -> runtime.javaVersion == javaVersion)
+                .findFirst().orElse(new Runtime(operatingSystem, webContainer, javaVersion));
+    }
+
+    public static Runtime getRuntimeFromWebApp(WebAppBase webAppBase) {
```

Comment:
I think this should be webapp's responsibility, i.e. `webapp.getRuntime()`

Reply:
Yes, will move the conversation to Utils for managers

Reply:
Yes, will move the codes to convertation utils